### PR TITLE
Add skipto navigation

### PR DESCRIPTION
**Pull request by marabooy**
This solves an accessibility issue on the site for people using screen readers and keyboard

Repro Steps: ​​​
1)Hit the URL "https://www.odata.org/"  to open Odata website​
2)Tab till 'Developers' link and press enter​
3)Tab till 'Getting Started' option and press enter​
4)Verify whether there is option to skip the repetitive content present in the page or not​
​
Actual Result: ​​​
There is no option to skip the repetitive content in the application (there is no skip link for the application)​
​
Expected Result:​
User should be provided with skip link so that one can skip the repetitive content and navigate directly to the main content​
​
User Impact: ​​
Keyboard users and screen reader users will find it difficult to navigate through the repetitive content every time to access the main content if there is no skip links​

Fix:
For users who are using keyboard this allows them to skip to any navigation element and content in the page using https://github.com/paypal/skipto. 

The menu highlights only on tab.
image

And brings this as a map when the user expands the menu for skipto
image

### CODE REVIEW

**Comment by habbes** [comment]
@marabooy could you link the issue this PR fixes, or create one if there isn't any? One that explains the problem.

File: _includes/scripts.html [hunk 1]
```diff
@@ -21,4 +21,29 @@
...
+			"attachElement": ".skip-nav-elements" // or "attachElement": "#MyCustomId"
+		}
+	}
+};
+</script>
+
+<script src="/public/skipto.min.js" type="text/javascript"></script>
+
+<script>
+	skipToMenuInit(SkipToConfig)
```
[the remark this answers is not in the document]
Comment (marabooy) [reply _includes/scripts.html]: the library looks for the object after loading. removed the other code js tag that did the same

File: _includes/scripts.html [hunk 2]
```diff
@@ -21,4 +21,29 @@
 ga('create', 'UA-55977038-1', 'auto');
 ga('send', 'pageview');
 </script>
--->
\ No newline at end of file
+-->
+
+
+<script >
+var SkipToConfig =
```
[the remark this answers is not in the document]
Comment (marabooy) [reply _includes/scripts.html]: @g2mula Wontfix as the library looks for this global object during instantiation

File: _includes/scripts.html [hunk 3]
```diff
@@ -21,4 +21,29 @@
...
+			"accesskey"    : "0",
+			"wrap"         : "true",
+			"visibility"   : "onfocus",
+			"attachElement": ".skip-nav-elements" // or "attachElement": "#MyCustomId"
+		}
+	}
+};
+</script>
+
+<script src="/public/skipto.min.js" type="text/javascript"></script>
```
[the remark this answers is not in the document]
Comment (marabooy) [reply _includes/scripts.html]: changed

**Pull request merged**